app/src/organisms/Devices/ProtocolRun/RunFailedModal.tsx

```diff
@@ -43,13 +43,15 @@ interface RunFailedModalProps {
   runId: string
   setShowRunFailedModal: (showRunFailedModal: boolean) => void
   highestPriorityError?: RunError | null
+  commandErrorList?: RunCommandErrors | null
```

Comment:
Somewhat nitty, but I strongly feel that we should avoid passing around `undefined` when we can avoid it (yeah, the existing code does it directly above yours 😅 ). We lose type-checking with `undefined` params when using the interface, so others who use the component might think it's ok avoid passing it in and still have correct behavior.

Reply:
So you prefer this to be mandatory? Ist it ok to avoid passing in commandErrorList? and get highestPriorityError instead?

Reply:
I mean something more like this:

```suggestion
  commandErrorList: RunCommandErrors | null
```